### PR TITLE
Added GitHub labels into the Contributor Guide

### PR DESCRIPTION
**Pull request by tradej**
Please have a look and provide feedback. Once this PR is in, I will create the labels accordingly.

### CODE REVIEW

File: docs/topics/contributing-labels.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+[options="header", cols="1,3a"]
+|===
+| Label | Description
+
+| WildFly Swarm | The issue or pull request relates to the WildFly Swarm runtime.
+| Spring Boot | The issue or pull request relates to the Spring Boot runtime.
+| Vert.x | The issue or pull request relates to the Eclipse Vert.x runtime.
+|===
+
+.Mission Labels
```
Comment (rhoads-zach) [inline docs/topics/contributing-labels.adoc]: Minor nitpick: can you make the label column link to the appropriate mission wiki listed here: https://github.com/openshiftio/appdev-documentation/wiki

Comment (tradej) [inline docs/topics/contributing-labels.adoc]: Sure, no problem. On it.

Comment (tradej) [inline docs/topics/contributing-labels.adoc]: @rhoads-zach, I have tried making links out of the label names, and the table doesn't look very good. I put the links in the description instead. I have also fixed the mission names, so it should be good to merge now if you're happy with it.

Comment (rhoads-zach) [inline docs/topics/contributing-labels.adoc]: LGTM. thank you @tradej

**Comment by rhoads-zach** [comment]
merged: d0c744194be800372aff0d35ac005f3a99e31609

**Pull request closed without merging**